seacatauth/otp/service.py

```diff
@@ -167,10 +179,24 @@ async def _delete_expired_totp_secrets(self):
 				"count": result.deleted_count
 			})
 
+	async def has_activated_totp(self, credentials_id: str) -> bool:
+		"""
+		Check if the user has TOTP activated from TOTPCollection. (For backward compatibility: check it from credentials (the old way))
+		"""
+		collection = self.StorageService.get(collection=self.TOTPCollection, obj_id=credentials_id)
```

Comment:
`credentials = await self. ....`
you're not getting a collection here. you are getting a credentials **object from** a credentials **collection**.
also, the get() method is a coroutine, which needs to be `await`ed.